lib/tasks/after_tasks.rake

```diff
@@ -572,8 +572,8 @@ namespace :After do
       end
       if chapter.content.match /<(embed|iframe)/
         begin
-          chapter.content_sanitizer_version = -1
-          chapter.sanitize_field(chapter, :content)
+          content = chapter.sanitize_value(:content, chapter.content)
+          chapter.update_attribute(:content, content)
```

Comment:
We generally keep them around in case someone needs to update an older instance of the site (admittedly probably pretty rare). Your fixes for the tasks look good, the tasks can stay.

Reply:
Since the three tasks are centered around resanitizing chapters, you can actually update an older instance of the site by running `rake resanitize:all` instead of the three individual tasks. But I've already done the work of updating them, so I don't mind leaving them in.